factor/gmp-ecm/tinyecm.c fails to build when USE_AVX512F is not set

```
factor/gmp-ecm/tinyecm.c:5217:3: error: too few arguments to function ‘tecm_dispatch’
 5217 |   tecm_dispatch(n, f, target_bits, pran);
      |   ^~~~~~~~~~~~~
factor/gmp-ecm/tinyecm.c:4667:5: note: declared here
 4667 | int tecm_dispatch(mpz_t n, mpz_t f, int targetBits, int arbitrary, uint64_t* ploc_lcg)
      |     ^~~~~~~~~~~~~
```

Broken in commit c1ad4630e5589be31bc02cd62a6472c531f5718e
Line 5217 is only activated when USE_AVX512F is not set, which will explain how the commit got through the testing

--- a/factor/gmp-ecm/tinyecm.c
+++ b/factor/gmp-ecm/tinyecm.c
@@ -5214,7 +5214,7 @@ void getfactor_tecm_x8_list(uint64_t* q64, uint64_t* f64, int target_bits,
 	for (i = 0; i < num_in; i++)
 	{
 		u128_to_mpz(q64 + 2 * i, n);
-		tecm_dispatch(n, f, target_bits, pran);
+		tecm_dispatch(n, f, target_bits, 1, pran);
 		mpz_to_u128(f, f64 + 2 * i);
 	}
 

